agent/xds/clusters.go

```diff
@@ -0,0 +1,79 @@
+package xds
+
+import (
+	"errors"
+	"time"
+
+	"github.com/gogo/protobuf/proto"
+
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2"
```

Comment:
`v2`, `auth`, `core` would probably read a little better in the body of the file if they packages imports were aliased to something like `v2->envoy`, `auth->envoyauth`, `core->envoycore`